src/main/java/kr/codesqaud/cafe/repository/ArticleRepository.java

```diff
@@ -0,0 +1,36 @@
+package kr.codesqaud.cafe.repository;
+
+import kr.codesqaud.cafe.domain.Article;
+import org.springframework.stereotype.Repository;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class ArticleRepository {
+    private final List<Article> articleList; //이번에는 List로 해보기
```

Comment:
```suggestion
    private final List<Article> articleList = new ArrayList<>();
```

생성자에서 초기화하지 않으셔도 이렇게 명시적으로 초기화할 수 있습니다 :) 그렇다면 이 둘의 차이는 무엇일까요?

Reply:
오 이것은 ......! 제가 지난 리뷰 PR 당시에 바로 이 부분을 이해하지 못해서 말씀하신 명시적인 초기화로 했었습니다
(확실히 명시적인 초기화도 방법인데 여러 다른 분들의 코드에서 생성자 초기화만 보이는 것이 오히려 이상해서 그랬었는데요 ㅎㅎㅎ) 당시 리뷰어께서 코멘트 해주셔서 확인해 보니 
-> 우선 생성자로 초기화를 하는 것은  의존성 주입 방법 중 '생성자 주입' 이었는데요 


~~~~~!!? 아-앗.... 제가 지금 (또...) 질문주신 요점을 놓치고 있었던 것 같아서 부끄럽습니다 😅 💦 
그 둘의 차이는 무엇일지(!?) ---->     모르겠다는 느낌입니다(앗..ㅎㅎㅎㅎㅎㅎㅎ)

알아내서, 여기에 정리하도록 하겠습니다!!! 🙌 😆 🚀